frontend/src/components/Common/Breadcrumb.js

```diff
@@ -0,0 +1,39 @@
+import { React } from 'react'
+import { useLocation } from 'react-router-dom'
+import { Breadcrumb, BreadcrumbItem } from 'reactstrap'
+import { pathToPageName } from '../../routes'
+
+const Breadcrumbs = () => {
```

Comment:
Nit: Rename the file or the method name for consistency

Reply:
Done in [1863c90](https://github.com/NickPriv/FryRank/pull/21/commits/1863c901244d5475537db2b6e7b7465160bc4aad).